models.rb

```diff
@@ -0,0 +1,89 @@
+require 'date'
+
+class Person
+    attr_accessor :name, :job, :email, :bank_accounts
+  
+    def initialize(name, job, email)
+      @name = name
+      @job = job
+      @email = email
+      @bank_accounts = []
+    end
+  
+    def add_bank_account(account)
+      @bank_accounts << account
+    end
+  
+    def remove_bank_account(account)
+      @bank_accounts.delete(account)
+    end
+
+    def to_string
+        return "name: #{@name}, job: #{@job}, email: #{email}, bank_accounts: #{@bank_accounts.length}"
```

Comment:
No need to use `return` in Ruby in this case. Methods in Ruby automatically return the last row from the method. For example:
```ruby
def string
  something = "This is a string"

  something
end
```
,
```ruby
def string
  something = "This is a string"

  return something
end
```
,
```ruby
def string
  return "This is a string"
end
```
and
```ruby
def string
  "This is a string"
end
```

will all have the same output